tests/OmniSharp.Roslyn.CSharp.Tests/CustomRoslynAnalyzerFacts.cs

```diff
@@ -211,7 +209,7 @@ public async Task When_custom_rule_is_set_to_none_dont_return_results_at_all()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/66085")]
```

Comment:
The underlying issue here has been fixed (thanks Manish!), however the fix is in a newer version of Roslyn, so skipping the test for now